src/pages/calendar/components/AddTask/TaskModal.jsx

```diff
@@ -0,0 +1,234 @@
+import { useState } from 'react'
+import { Modal, Button } from '../../../../components'
+import styled, { css, useTheme } from 'styled-components'
+import { PlusIcon } from '../../../../assets/icons'
+import { useTranslation } from 'react-i18next'
+import { LocalizationProvider, MobileTimePicker } from '@mui/x-date-pickers'
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs'
+import dayjs from 'dayjs'
+import { RadioButton } from './RadioButton'
+const priority = ['low', 'medium', 'high']
+
+export const TaskModal = ({ date,visible,setVisible }) => {
+  const { t } = useTranslation()
+  const [title, setTitle] = useState('')
+  const [startDate, setStartDate] = useState(dayjs(date).unix())
+  const [endDate, setEndDate] = useState(dayjs(date).unix())
+  const [selectedPriority, setSelectedPriority] = useState('low')
+  const {colors} = useTheme()
+
+  const handleTitleChange = (event) => {
```

Comment:
looks like you didn't connect formik to your states. Please connect it

Reply:
I am not using formik.